qt/applications/workbench/workbench/plugins/logmessagedisplay.py

```diff
@@ -14,12 +14,13 @@
 
 # 3rdparty imports
 from mantidqt.utils.writetosignal import WriteToSignal
+from mantidqt.utils.qt import toQSettings
```

Comment:
are `mantidqt` imports not local imports? or by local do we mean relative path `..`?
`codeeditor/editor.py`, also in this PR, has it under local imports

Reply:
I count local to mean the top-level package that we are in, i.e workbench, and then things we use, i.e mantidqt, as 3rd party. So `mantidqt` is an outside package to workbench for example. I'll double check those imports. PyCharm has a habit of reorgainzing them slightly differently dependent on the platform.

Reply:
In `codeeditor/editor.py` we are in `mantidqt` so that one I think counts as a local import in that context.